README.md

```diff
@@ -1,4 +1,5 @@
-![LOGO](resources/logo-medium.png)
+![LOGO](https://cloud.githubusercontent.com/assets/762949/15234285/e84cc032-1864-11e6-9fb5-ced867249343.png)
```

Comment:
This way, we don't need to relatively reference the logo if someone opens it locally without `resources` directory.